src/ResponseTagger/Delegator/XLocationIdTagger.php

```diff
@@ -0,0 +1,73 @@
+<?php
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license   For full copyright and license information view LICENSE file distributed with this source code.
+ */
+
+namespace EzSystems\PlatformHttpCacheBundle\ResponseTagger\Delegator;
+
+use EzSystems\PlatformHttpCacheBundle\ResponseConfigurator\ResponseCacheConfigurator;
+use EzSystems\PlatformHttpCacheBundle\ResponseTagger\ResponseTagger;
+use Symfony\Component\HttpFoundation\Response;
+use eZ\Publish\API\Repository\Repository;
+
+class XLocationIdTagger implements ResponseTagger
+{
+    /**
+     * @var Repository
+     */
+    private $repository;
+
+    /**
+     * @var ResponseTagger
+     */
+    private $contentInfoTagger;
+
+    /**
+     * @var ResponseTagger
+     */
+    private $locationTagger;
+
+    const LOCATION_ID_HEADER = 'X-Location-Id';
+
+    /**
+     * XLocationIdTagger constructor.
+     *
+     * @param                Repository $
+     * @param                $repository
+     * @param ResponseTagger $contentInfoTagger
+     * @param ResponseTagger $locationTagger
+     */
+    public function __construct(
+        Repository $repository,
+        ResponseTagger $contentInfoTagger,
+        ResponseTagger $locationTagger
+    ) {
+        $this->repository        = $repository;
+        $this->contentInfoTagger = $contentInfoTagger;
+        $this->locationTagger    = $locationTagger;
```

Comment:
I get what you are trying to do here, but let's just map the location id to xkey equivalent here.

PR's like #7 and similar are meant to add the other tags based on the data already loaded, to avoid us having to load repository objects several times.

In other words, you won't need any dependencies here.